torch/csrc/distributed/rpc/py_rref.h

```diff
@@ -13,7 +13,9 @@ namespace rpc {
 class PyRRef {
  public:
   explicit PyRRef(const py::object& value, const py::object& type_hint);
-  explicit PyRRef(c10::intrusive_ptr<RRef> rref);
+  explicit PyRRef(
+      c10::intrusive_ptr<RRef> rref,
+      const std::shared_ptr<FutureMessage> fm = nullptr);
```

Comment:
Currently it can be null in `_invoke_remote_torchscript` in rpc/init.cpp. I didn't add the support there since this PR wasn't meant to add all of the JIT supported. We are waiting on the consolidation of the futures in order to do that properly.